contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -3992,8 +3992,33 @@ get_start_token_of_batch_level_stmt_body(TSqlParser::Batch_level_statementContex
 	return nullptr;
 }
 
+char*
+createHandlingForBatchLevelStatement(TSqlParser::Batch_level_statementContext *ctx, const char *originalQuery)
```

Comment:
Please rename it to something meaningful like `storeOriginalQueryForBatchLevelStatement`.